src/containers/modal-wheel/index.js

```diff
@@ -0,0 +1,289 @@
+/* --------------------------------------------------------
+* Author Võ Bách Nhạc
+* Email vonhac.20394@gmail.com
```

Comment:
Please set author, your number phone,
remove console.log in this file

Reply:
fixed